site/content/api/Command-And-Control-API.md

```diff
@@ -12,43 +12,39 @@ THIS IS A FIRST DRAFT VERSION OF THE API SPEC AND THUS STILL SUBJECT TO DISCUSSI
 IF YOU THINK THAT THE SPEC SHOULD BE CHANGED OR CAN BE IMPROVED PLEASE DO NOT HESITATE TO OPEN AN ISSUE OR DROP US A LINE ON THE MAILING LIST.
 {{% /warning %}}
 
-A client can be a device connected to Hono directly (using an AMQP connection) or via a *Protocol Adapter* (for example via MQTT,
-LWM2M, etc). Devices and *Protocol Adapters* can also poll Hono periodically in order check for new command & control messages without
-maintaining a constant connection to Hono infrastructure.
+The Command & Control API is defined by means of AMQP 1.0 message exchanges, i.e. a client needs to connect to Hono using AMQP 1.0 in order to invoke operations of the API as described in the following sections. Throughout the remainder of this page we will simply use AMQP when referring to AMQP 1.0.
 
-The Command & Control API is defined by means of AMQP 1.0 message exchanges, i.e. a client needs to connect to Hono using an AMQP 1.0 client in 
-order to invoke operations of the API as described in the following sections.
+# Operations
 
-# Southbound Operations
+The following API defines operations that can be used by *Business Applications* to send control messages to *Protocol Adapters* (and by *Protocol Adapters* to receive them) as well as the response from the *Protocol Adapters* back to the *Business Applications*.
 
-The following operations can be used by *Devices* and/or *Protocol Adapters* to receive command & control messages sent by *Business Applications*.
-
-## Receive Control Message
+## Send and Receive a Command
```

Comment:
If this is written from the perspective of the actor (the application) then it should probably just be *Send a Command*.